RELEASES.md

```diff
@@ -9,11 +9,17 @@
 
 * Update beats
 
-  run `make update-beats` in the branch from which the new branch will be created before FF to recognize potential issues
+  * Before feature freeze and creating the release branch, update `BEATS_VERSION` to the release version in the top-level Makefile of the release's parent branch
```

Comment:
I'm not sure I understand this addition. Maybe some examples would help.

In this case for example, let's say the release is 7.14. I take "the release's parent branch" to mean 7.x. Is that what you meant? We shouldn't need to update `BEATS_VERSION` in the 7.x branch - that's only updated when we branch, not before. This is part of the `update versions in release branch` task below, but it could probably be called out.

I think the idea of the existing comment was that if we (ideally regularly) run `make update-beats` on the 7.x branch, we _should_ avoid surprises when it comes to time to branching and updating beats on the release branch. In the hopefully-not-too-distant-future this should be taken care of by https://github.com/elastic/apm-server/issues/4941

Reply:
ah k. yeah I thought this was essentially a repeat of the below step, where this gets run regularly before a "final"-ish update post-branch.

I'll remove this section then?

Reply:
Yeah I think so. Maybe call out the BEATS_VERSION change in the below section?

Reply:
cool, updated